app/middlewares/authCheck.js

```diff
@@ -1,15 +1,21 @@
 const errors = require('../errors'),
   User = require('../models').user,
+  time = require('time'),
   sessionsManager = require('../services/sessionsManager.js');
 
+const MILISECONDS_PER_MINUTE = 60000;
+
 exports.checkUser = (req, res, next) => {
   const auth = req.headers.authorization; // auth is in base64(username:password)  so we need to decode the base64
   try {
     const decoded = sessionsManager.decode(auth);
+    const currentTime = new Date();
+    const minutesSinceLogin = Math.floor((currentTime.getTime() - decoded.exp.time) / MILISECONDS_PER_MINUTE);
+    if (minutesSinceLogin > 1) throw errors.expiredSession;
```

Comment:
you shouldn't have magic numbers in your code, specially when the card tells that the idea is to be easily configurable.
If the client requires the expiration time to be 60 minutes, with the current solution you to change this magic number, which implies that the one changing it knows how the app works and where to look.
There are several changes that can be done to improve this situation, for example:
1- Move this magic number to a constant with a descriptive name.
2- Move this magic number to a configuration variable in `config/index.js`
3- Same as number 2 and populate the configuration variable in `config/index.js` with an environment variable.
4- Make a db table to represent the configuration and implementing some endpoints to be able to show and edit that configuration. 

The best thing to make in the training would be number 3, because number 4 is the best but implies a lot of work. If you want to talk a little bit more about it let me know and we talk via slack.